spec/requests/api/v1.1/portfolios_controller_spec.rb

```diff
@@ -145,7 +145,7 @@
     let(:params) do
       {:group_uuids => [group_uuid], :permissions => ["read"]}
     end
-    let(:share_resource) { instance_double(Catalog::ShareResource) }
+    let(:share_resource) { instance_double(Api::V1x0::Catalog::ShareResource) }
```

Comment:
Shouldn't this be `Api::V1x1` since we're in `spec/requests/api/v1.1`? Same goes for the rest of them in this file.